Sources/SendGridKit/Models/AdvancedSuppressionManager.swift

```diff
@@ -2,21 +2,25 @@ import Foundation
 
 public struct AdvancedSuppressionManager: Codable, Sendable {
     /// The unsubscribe group to associate with this email.
-    public var groupId: Int
+    /// 
+    /// See the Suppressions API to manage unsubscribe group IDs.
+    public var groupID: Int
     
     /// An array containing the unsubscribe groups that you would like to be displayed on the unsubscribe preferences page.
+    /// 
+    /// This page is displayed in the recipient's browser when they click the unsubscribe link in your message.
     public var groupsToDisplay: [String]?
     
     public init(
-        groupId: Int,
+        groupID: Int,
```

Comment:
These are breaking changes, do we want to have a new major version?

Reply:
There are other **required** breaking changes I made to fully comply with SendGrid API (and more to come in other PRs), so yes, we will need a new major version down the line